views/peep.erb

```diff
@@ -0,0 +1,16 @@
+<form action= '/new-peep' method='post'>
+<h1> Enter a Peep </h1>
+  <input type= 'text' name='peep'>
+  <br>
+  <br>
+  <input type= 'submit' value='Peep'>
+</form>
+
+<ul>
+<% @peeps.each do |peep| %>
+<li class='peep' >
+  <%= '#{peep.peep}' %>
+  <%= '#{peep.time}' %>
```

Comment:
When I ran rackup, the localhost showed '#{peep.peep}' and '#{peep.time}' which was strange because this usually works so it could just be something on my end. I changed the single quotes to double quotes and it worked.